firmware/raspi/boardinterface.py

```diff
@@ -9,6 +9,10 @@
 
 from status import ArduinoException, ArduinoStatus, OpCode
 import util
+import serial
+
+#Need to find the pi port and settings we intend to use
```

Comment:
This should be a `# TODO: blah` type comment so that we know to come back to it later.